compose/ui/ui-text/src/skikoMain/kotlin/androidx/compose/ui/text/platform/PlatformFont.skiko.kt

```diff
@@ -196,14 +196,14 @@ class FontLoadResult(val typeface: SkTypeface?, val aliases: List<String>)
 
 internal class FontCache {
     internal val fonts = FontCollection()
-    private val fontProvider = TypefaceFontProvider()
+    private val fontProvider = TypefaceFontProvider.createAsFallbackProvider()
     private val registered: MutableSet<String> = HashSet()
     private val typefacesCache: Cache<String, SkTypeface> = ExpireAfterAccessCache(
         60_000_000_000 // 1 minute
     )
 
     init {
-        fonts.setDefaultFontManager(FontMgr.default)
+        fonts.setDefaultFontManager(FontMgr.defaultWithFallbackFontProvider(fontProvider))
```

Comment:
It affects all targets.
In most cases desktop and ios won't be affected, because the default font manager can provide the fallbacks, and therefore the new logic in skiko won't be used.